wagtail_webstories/models.py

```diff
@@ -42,6 +43,7 @@ class WebStoryPageMixin(models.Model):
     PORTRAIT_IMAGE_FILTER = 'fill-640x853'
     SQUARE_IMAGE_FILTER = 'fill-640x640'
     LANDSCAPE_IMAGE_FILTER = 'fill-853x640'
+    extra_args = {"use_json_field": True} if WAGTAIL_VERSION < (6, 0) else {}
```

Comment:
This isn't really needed - Wagtail will continue accepting the `use_json_field` argument, even if it no longer has any effect, since it will still be present in historical migrations (and we don't expect developers to go through those migrations removing it). My recommendation would be to leave the `use_json_field=True` in place for now, and remove it when we drop support for Wagtail <6.0.

You _could_ make an argument that making this change now will make it easier to spot it when that removal happens... however, the corresponding change to `0005_external_story_blocks.py` worries me a bit because it's adding conditional code to a migration, and it's hard to verify that this is going to be correct in all cases. So, I think the problems outweigh the benefits, and I would suggest reverting d150a71e2dc6b9accd8430cc58cb0600d88ab7e7.

Reply:
Thanks, for reviewing @gasman .
Reverted that commit.